src/data/int/parity.lean

```diff
@@ -27,12 +27,6 @@ local attribute [simp] -- euclidean_domain.mod_eq_zero uses (2 ∣ n) as normal
 theorem mod_two_ne_zero : ¬ n % 2 = 0 ↔ n % 2 = 1 :=
 by cases mod_two_eq_zero_or_one n with h h; simp [h]
 
-@[simp] theorem even_coe_nat (n : ℕ) : even (n : ℤ) ↔ even n :=
```

Comment:
I moved this because I wanted to put it close to `odd_coe_nat`, but my proof relies on lemmas below this.